README should contain install instructions

When running `make run` on a fresh machine I get:

``` bash
$ make run
docker-compose up -d
#ERROR: Couldn't connect to Docker daemon. You might need to install Docker:

http://docs.docker.io/en/latest/installation/ubuntulinux/
Makefile:45: recipe for target 'run' failed
make: *** [run] Error 1
```

There should be instructions on how to set up the full environment in the README file.

--- a/README.md
+++ b/README.md
@@ -35,11 +35,7 @@ Requirements
 Notes
 ----
 
-If you get an ubuntu-vanilla-theme error on first startup, go into your /node_modules/ubuntu-vanilla-theme directory and run:
-
-``` npm install --save-dev gulp-sass@2 ```
-
-then
+If you get an ubuntu-vanilla-theme error on first startup, try the following:
 
 ``` docker-compose run sass ```
 

--- a/README.md
+++ b/README.md
@@ -23,12 +23,29 @@ Now visit <http://127.0.0.1:8001>
 
 To see what other `make` commands are available, run `make help`.
 
+Requirements
+----
+
+* **Docker** - you will need to download, install and run docker
+* **npm**
+  * Install the node.js package manager, npm, this might be easier with the node version manager,  [nvm]](https://github.com/creationix/nvm/blob/master/README.markdown). On a Mac with homebrew, try `brew install nvm` and follow the instructions, then `nvm install --lts` to install the long term supported version of node
+  * Rebuild the sass with `docker-compose run sass`
+  * Restart the server `make stop run`
+
 Notes
 ----
 
 If you get an ubuntu-vanilla-theme error on first startup, go into your /node_modules/ubuntu-vanilla-theme directory and run:
 
 ``` npm install --save-dev gulp-sass@2 ```
+
+then
+
+``` docker-compose run sass ```
+
+and even possibly
+
+``` make stop run ```
 
 License
 ---

--- a/README.md
+++ b/README.md
@@ -26,9 +26,9 @@ To see what other `make` commands are available, run `make help`.
 Requirements
 ----
 
-* **Docker** - you will need to download, install and run docker
+* **Docker** - you will need to download, install and run [docker](https://www.docker.com/products/docker)
 * **npm**
-  * Install the node.js package manager, npm, this might be easier with the node version manager,  [nvm]](https://github.com/creationix/nvm/blob/master/README.markdown). On a Mac with homebrew, try `brew install nvm` and follow the instructions, then `nvm install --lts` to install the long term supported version of node
+  * Install the node.js package manager, [npm](https://nodejs.org/en/download/), this might be easier with the node version manager,  [nvm]](https://github.com/creationix/nvm/blob/master/README.markdown). On a Mac with homebrew, try `brew install nvm` and follow the instructions, then `nvm install --lts` to install the long term supported version of node
   * Rebuild the sass with `docker-compose run sass`
   * Restart the server `make stop run`
 

